SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialVirtualWorkerTranslator.h

```diff
@@ -2,23 +2,24 @@
 
 #pragma once
 
-#include "CoreMinimal.h"
-
-#include "Containers/Queue.h"
+#include "EngineClasses/AbstractVirtualWorkerTranslator.h"
 #include "SpatialCommonTypes.h"
 #include "SpatialConstants.h"
 
 #include <WorkerSDK/improbable/c_worker.h>
 #include <WorkerSDK/improbable/c_schema.h>
 
+#include "Containers/Queue.h"
```

Comment:
What is this being used for?

Reply:
`	TQueue<VirtualWorkerId> UnassignedVirtualWorkers;`

Reply:
I can't see a TQueue in this header. Also are you sure TQueue is the correct type?